deploy/complete/kubernetes/README.md

```diff
@@ -11,8 +11,26 @@ Refer to [helm-chart](../helm-chart/README.md)
 > NOTE: There are several options for ingress controllers in K8S. This demo uses the common `ingress-nginx`
 
 ```text
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/cloud-generic.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/mandatory.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/provider/cloud-generic.yaml
+```
+
+```bash
+kubectl -n ingress-nginx get svc
+```
+
+### Enable the monitoring capability by adding your OCI credentials:
+
+```bash
+kubectl create secret generic monitoring-secret \
```

Comment:
Create a OCI secret need to be optional, as we support deploy MuShop on any platform and in mock mode, not needing any extra configuration.

Reply:
It is optional, however will need to change mushop.yaml to mushop-monit-scenario.yaml and a new README if we decide to go ahead with this implementation. Can remove these steps from README and just add it on the scenario.md would also help avoid confusions.

Reply:
"monit" sounds strange, and do not need to specify that is a "Scenario". Use mushop_with_monitoring.yaml or something more verbal.